### ISSUE TITLE
Cannot stream live TV with TVHeadend plugin, when authentication is active.

### ISSUE THREAD

**Issue by dalgwen** [jellyfin/jellyfin-plugin-tvheadend#10]
**Describe the bug**
Cannot stream live TV with TVHeadend plugin, when authentication is active.
When I disable authentication in TVHeadend, playing is OK.
When I play recordings, even with authentication, playing is OK.

**To Reproduce**
1. Have a jellyfin server connected to TVHeadend with mandatory login/password authentication 
2. Try to play a live TV channel
3. Does not play

**Expected behavior**
Live TV channels should play fine, as recordings do.

**Logs**
When I try to play Live TV, in the log I see the ffmpeg command launched :
`/usr/lib/jellyfin-ffmpeg/ffmpeg -analyzeduration 3000000 -i "http://192.168.0.1:9981/stream/channelid/1156837874?ticket=D430E774C45AF2D0A141D1A62C59FB022A1309CF" -map_metadata -1 -map_cha [....]`
---> There is no login password in the command line

FYI, here is the ffmpeg command launched  when I play a recording :
`/usr/lib/jellyfin-ffmpeg/ffmpeg -f mpegts -i "http://login:password@192.168.0.1:9981/dvrfile/788633301" -map_metadata -1 -map_chapters -1 -threads 0 -map 0:0 [...]`
--> You could notice the login / password field is there this time.

**System (please complete the following information):**
 - OS: Debian
 - Browser: Firefox
 - Jellyfin Version: [e.g. 10.3.2]
 - Reverse proxy: nginx

**Additional context**
Many, many thanks for the great work !

**Comment by ggogel** [comment]
You need to enable plain authentication:
Configuration / General / Base / HTTP Server Settings / Authentication type: Both plain and digest

**Comment by dalgwen** [comment]
Oh, I completely forgot this ticket !

I just did another test today, and this time it works.
I guess it's the new tvheadend installation I did last year. I probably fixed some misconfiguration without even knowing it.
The command line is the same, with no login/password. But since it is working, I suppose the "ticket" parameter is enough for the authentication, and I was wrong to point this as a possible cause.

Case closed, sorry for bothering you.
